### ISSUE TITLE
Mount host dir to guest vm

### ISSUE THREAD

**Issue by johndo100** [yuichiro-naito/bmd#8]
I want to know how to mount host directory to guest vm.

Please help.

**Comment by yuichiro-naito** [comment]
One way is to use network file systems such as NFS or SAMBA. These protocols may work for you if your VMs have network connectivity to the host OS.
If you intend to use a virtio-9p device, the bmd doesn't support it now. I'm planning to add the configurations for virtio-9p. The guest FreeBSD is now able to use it by the following commit. It looks so fun.

https://cgit.freebsd.org/src/commit/?id=e97ad33a89a78f55280b0485b3249ee9b907a718

**Comment by johndo100** [comment]
Yes. Virtio-9p is exactly what I use to mount host dir when I'm on byhve-vm. Jail manager apps usually use something call nullfs but I'm not sure. Sorry my knowledge about it is little.

**Comment by yuichiro-naito** [comment]
I added 'sharefs' keyword to the VM configuration.
It has "[sharename] = [pathname]" value which is passed to the virtio-9p device.

I'm feeling sharing a file system is not a block device. Of course, virtio-9p is one of PCI device of the VM,
but it's an implementation matter.  For VM users, it looks easy to understand that 'disk' is used for block devices,
and 'sharefs' is used for file systems.

https://github.com/yuichiro-naito/bmd/commit/bdf1864d2c3a742b8236ece9d00a510be700fc9c

This feature will be included in the next release.

**Comment by yuichiro-naito** [comment]
This feature was released in version 3.1. Now the FreeBSD ports tree has version 3.2.
I'll close this issue.
